### PR TITLE
Added priority queue scheduling algorithm Fixes: #2510

### PR DESCRIPTION
**Pull request by verdantfire**
### **Describe your change:**
Implementation of Priority Queue scheduling algorithm. In this Algorithm we only care about the order that the processes arrived without caring about their duration time. 
Wikipedia: [Priority Queue](https://en.wikipedia.org/wiki/Scheduling_(computing)#Priority_scheduling)

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: scheduling/priority_queue.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
+# Implementation of Priority Queue scheduling algorithm (without preemption)
+# In this Algorithm we only care about the order that the processes arrived
+# without caring about their duration time
+# https://en.wikipedia.org/wiki/Scheduling_(computing)#Priority_scheduling
+from __future__ import annotations
+
+
+def sort_processes_by_priority(
+    processes: list[int], duration_times: list[int], priorities: list[int]
+) -> list[int]:
```
Comment (poyea) [inline scheduling/priority_queue.py:10]:
```suggestion
    processes: List[int], duration_times: List[int], priorities: List[int]
) -> List[int]:
```

Comment (verdantfire) [reply scheduling/priority_queue.py:10]: It was due to the doctests in a function.

File: scheduling/priority_queue.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    sorted_processes, sorted_duration_times = [], []
+    for process, duration_time, priority in zip(processes, duration_times, priorities):
+        prioritized_queue.append((process, duration_time, priority))
+    prioritized_queue = sorted(prioritized_queue, key=lambda x: (x[2]))
+    priorities = priorities.sort()
+
+    for process, duration_time, priority in prioritized_queue:
+        sorted_processes.append(process)
+        sorted_duration_times.append(duration_time)
+    return sorted_processes, sorted_duration_times
```
Comment (sunnygupta3535) [inline scheduling/priority_queue.py:32]: Syntax Error: 'return' is outside of a function

Comment (sunnygupta3535) [inline scheduling/priority_queue.py:32]: Syntax Error: 'return' is called outside of a function

**Review by sunnygupta3535 — CHANGES_REQUESTED** [review]
At Line number 32: return is called outside of function.

**Pull request closed without merging**